### ISSUE TITLE
Not supported plan in JSON format

### ISSUE THREAD

**Issue by kino505** [Bjorn248/terraform_cashier#18]
I didn't find a possibility of processing plan files in JSON format. What a pity. Is it hard to realize?

**Comment by Bjorn248** [comment]
Correct. Currently, only bson plans are supported. I didn't know it was possible to use `terraform plan` to generate a JSON format plan file. I can't find the option anywhere on [this page](https://www.terraform.io/docs/commands/plan.html).

**Comment by kino505** [comment]
Oh, it is not direct path:)
terraform plan -out=plan.save
terraform show -json plan.save

**Comment by Bjorn248** [comment]
Ah, no, that is currently not supported. Only the output of `terraform plan` is supported.

**Comment by kino505** [comment]
Ok. I understood. Thank You for your answer. It will be useful when using several dedicated modules in a sequence (one by one).
